Formula/libirecovery.rb

```diff
@@ -0,0 +1,36 @@
+class Libirecovery < Formula
+  desc "Library to communicate with iDevices restore and recovery"
+  homepage "https://www.libimobiledevice.org/"
+
+  stable do
+    url "https://github.com/libimobiledevice/libirecovery/archive/0.1.1.tar.gz"
+    sha256 "81aab17730ebbbe598f91e8dd4249b76c1f0d6782c60061e59d3c52637b5305c"
+  end
+
+  head do
+    url "https://git.libimobiledevice.org/libirecovery.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+    depends_on "libxml2"
```

Comment:
Does the one from macOS not work?

Reply:
Testing with macOS - this didn't surface in audit or test.  Apologies

Reply:
It doesn't depend on libxml2 at all...